pkg/logs/config/config.go

```diff
@@ -60,3 +67,29 @@ func buildLogSources(ddconfdPath string, collectAllLogsFromContainers bool, tcpF
 
 	return logSources, nil
 }
+
+// buildServerConfig returns the server config to send logs to.
+func buildServerConfig() (*ServerConfig, error) {
+	switch {
+	case LogsAgent.GetString("logs_config.logs_dd_url") != "":
```

Comment:
Same here, I'd like to make it explicit that we're in the proxy mode.
It is very important, because when we're in proxy mode we will want to never ever update the logic.
However, when we're not in proxy mode but using `dd_url` to submit to a custom intake endpoint (myorg.datadoghq.com), we'll want to automatically update when the intake gets improvements